### ISSUE TITLE
Building is broken on Flutter beta channel

### ISSUE THREAD

**Issue by lukehutch** [jb3rndt/PersistentBottomNavBarV2#129]
If I switch to the Flutter beta channel, I get the following error message when I depend upon `flutter_persistent_bottom_nav_bar_v2`:

```console
../../../.pub-cache/hosted/pub.dev/persistent_bottom_nav_bar_v2-4.2.8/lib/neumorphic-package-by-serge-software/neumorphic-card.dart:57:57:
Error: The getter 'backgroundColor' isn't defined for the class 'ThemeData'.
 - 'ThemeData' is from 'package:flutter/src/material/theme_data.dart' ('/opt/flutter/packages/flutter/lib/src/material/theme_data.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'backgroundColor'.
    final color = decoration.color ?? Theme.of(context).backgroundColor;
                                                        ^^^^^^^^^^^^^^^
```

so I guess the Material API is changing, or something...

**Comment by jb3rndt** [comment]
Seems so. It is fixed with the next version :)

**Comment by jb3rndt** [comment]
This is fixed with version [5.1.0](https://pub.dev/packages/persistent_bottom_nav_bar_v2)
